docs/getting-started.md

```diff
@@ -120,15 +120,9 @@ command line options as follows (run a specified suite):
     FUNC_TEST_ARGS='-ginkgo.focus=vm_networking_test -ginkgo.regexScansFilePath' make functest
 ```
 
-In addition, if you want to run a specific test or tests you can prepend any Describe,
-Context and It statements of your test with an `F` and again use `gingko` command line
-options to run only tests marked with `F` (DON'T FORGET TO REMOVE BEFORE PUSHING):
-
-For example, to run the
-
-```
-    FUNC_TEST_ARGS='-ginkgo.regexScansFilePath' make functest
-```
+In addition, if you want to run a specific test or tests you can prepend any `Describe`,
+`Context` and `It` statements of your test with an `F` and Ginkgo will only run items
+that are marked with the prefix.
```

Comment:
I think the all-caps reminder not to commit `FIt` or other "focus" statements is worth keeping.